chain/gap/find_test.go

```diff
@@ -19,8 +19,8 @@ import (
 )
```

Comment:
changes here due to unify range flags (--to and --from) to use int64's. mechanical change